packages/iframe-stamper/src/index.ts

```diff
@@ -67,13 +67,7 @@ export class IframeStamper {
       );
     }
 
-    const container = document.getElementById(config.iframeContainerId);
-    if (!container) {
-      throw new Error(
-        `Cannot create iframe stamper: no container with ID ${config.iframeContainerId} exists in the current document`
-      );
-    }
-    this.container = container;
+    this.container = config.iframeContainer;
```

Comment:
I think the check above (on line 64) needs to be updated, from `document.getElementById(config.iframeElementId)` to ``this.container.querySelector(`#${config.iframeElement}`)``

Reply:
👀 👀  Fixed, thanks!